README.md

```diff
@@ -2,6 +2,15 @@
 
 ## Motivation
 
+## Installation
+ - For running TG code requirements_tg.txt can be used
+ - Install pgenlib from PLINK's repo (change accordingly)
+   - git clone https://github.com/chrchang/plink-ng.git
+   - cd plink-ng/2.0/Python
+   - /usr/local/bin/python setup.py build_ext
```

Comment:
It depends on the particular python setup, I'd suggest to write just `python3 setup.py build_ext` (accordingly to `setup.py`, python3 should be used).